### DISCUSSION TITLE
Is text-to-speech possible?

### DISCUSSION THREAD

**Discussion by dawoodjee** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#559] [Q&A]
I'm here because I wanted the same text-to-speech feature I get on my mobile when I ask google to

> "Read this article"

I'm on windows, not sure if this is possible from this project or not. Any ideas?

**Comment by im-coder-lg** [comment]
Well, not yet but if you know a npm module that can do the job, do tell us here!

**Reply by dawoodjee** [reply]
Thanks for the info. I currently don't. I think this could be useful

https://cloud.google.com/text-to-speech

**Reply by im-coder-lg** [reply]
Look, I really appreciate the help on tracking down the Text-to-speech part by Google. What I meant was, since this is a JS/Electron app, if there is a module that can be included in the file via JS, tell us about that and if you know how to add it, feel free to make a PR! Really appreciate the help, but we need a real module via npm.

**Reply by im-coder-lg** [reply]
Also, it has to be open-source so that the code can be inspected for malicious software and also on how to implement it.
